apps/condo/domains/ticket/schema/TicketChange.test.js

```diff
@@ -0,0 +1,344 @@
+            expect(objs[0].clientDisplayNameFrom).toEqual(client.user.name)
+            expect(objs[0].clientDisplayNameTo).toEqual(client2.user.name)
+
+            expect(objs[0].operatorIdFrom).toEqual(ticket.operator.id)
+            expect(objs[0].operatorIdTo).toEqual(payload.operator.connect.id)
+            expect(objs[0].operatorDisplayNameFrom).toEqual(client.user.name)
+            expect(objs[0].operatorDisplayNameTo).toEqual(client2.user.name)
+
+            expect(objs[0].assigneeIdFrom).toEqual(ticket.assignee.id)
+            expect(objs[0].assigneeIdTo).toEqual(payload.assignee.connect.id)
+            expect(objs[0].assigneeDisplayNameFrom).toEqual(client.user.name)
+            expect(objs[0].assigneeDisplayNameTo).toEqual(client2.user.name)
+
+            expect(objs[0].executorIdFrom).toEqual(ticket.executor.id)
+            expect(objs[0].executorIdTo).toEqual(payload.executor.connect.id)
+            expect(objs[0].executorDisplayNameFrom).toEqual(client.user.name)
+            expect(objs[0].executorDisplayNameTo).toEqual(client2.user.name)
+
+            expect(objs[0].classifierIdFrom).toEqual(ticket.classifier.id)
+            expect(objs[0].classifierIdTo).toEqual(payload.classifier.connect.id)
+            expect(objs[0].classifierDisplayNameFrom).toEqual(classifiers[0].name)
+            expect(objs[0].classifierDisplayNameTo).toEqual(classifiers[1].name)
+
+            expect(objs[0].sourceIdFrom).toEqual(ticket.source.id)
+            expect(objs[0].sourceIdTo).toEqual(payload.source.connect.id)
+            expect(objs[0].sourceDisplayNameFrom).toEqual(sources[0].name)
+            expect(objs[0].sourceDisplayNameTo).toEqual(sources[1].name)
+
+            expect(objs[0].relatedIdFrom).toEqual(ticket2.id)
+            expect(objs[0].relatedIdTo).toEqual(payload.related.connect.id)
+            expect(objs[0].relatedDisplayNameFrom).toEqual(ticket2.number.toString())
+            expect(objs[0].relatedDisplayNameTo).toEqual(ticket3.number.toString())
+
+            // TODO(antonal): figure out how to get old list of related items in many-to-many relationship.
+            expect(objs[0].watchersIdsFrom).toEqual(expect.arrayContaining([client2.user.id, client3.user.id]))
+            expect(objs[0].watchersIdsTo).toEqual(expect.arrayContaining([client3.user.id, client4.user.id]))
+            expect(objs[0].watchersIdsFrom.length).toEqual(2)
+            expect(objs[0].watchersIdsTo.length).toEqual(2)
+            expect(objs[0].watchersDisplayNamesFrom).toEqual(expect.arrayContaining([client2.user.name, client3.user.name]))
+            expect(objs[0].watchersDisplayNamesTo).toEqual(expect.arrayContaining([client3.user.name, client4.user.name]))
+            expect(objs[0].watchersDisplayNamesFrom.length).toEqual(2)
+            expect(objs[0].watchersDisplayNamesTo.length).toEqual(2)
+        })
+
+        it('not gets created when Ticket has no changes', async () => {
```

Comment:
you don't have a test that checks the value of from/to fields for non changed fields

Reply:
Will be implemented in https://doma.atlassian.net/browse/SBERDOMA-391